salalib/axialpolygons.cpp

```diff
@@ -35,15 +19,17 @@ AxialVertex AxialPolygons::makeVertex(const AxialVertexKey& vertexkey, const Poi
    // using an anglemap means that there are now no anti-clockwise vertices...
    // TODO: (CS) Double as key is problematic - books have been written about double equality...
    std::map<double,int> anglemap;
-   for (size_t i = 0; i < pointlist.size(); i++) {
-      anglemap.insert(std::make_pair( angle(openspace,av.m_point,pointlist[i]), i ));
+   int i = 0;
```

Comment:
If you need an index anyway, might just as well use `operator[]` on the vector - one thing less that can go out of sync.